build/gitversion.yml

```diff
@@ -1,6 +1,6 @@
 assembly-versioning-scheme: MajorMinorPatch
 mode: ContinuousDeployment
-next-version: 0.2.0
+next-version: 0.3.0 #TODO: confirm version
```

Comment:
We can probably make it 1.0.0. @jeanplevesque any objection?

Reply:
I think we could do a review pass before setting version 1.0